Sally7/ValueConversion/ToS7Conversions.cs

```diff
@@ -189,5 +180,46 @@ private static int ConvertFromString(string? value, int length, Span<byte> outpu
             return span.Length + 2;
 #endif
         }
+
+        private static int CopyBuffer(ReadOnlySpan<byte> input, Span<byte> output)
+        {
+            ref var destination = ref MemoryMarshal.GetReference(output);
+            ref var source = ref MemoryMarshal.GetReference(input);
+
+            var offset = 0u;
+            while (offset + sizeof(ulong) <= input.Length)
```

Comment:
```suggestion
            while (offset <= input.Length - sizeof(ulong))
```
`offset` is incremented at the end of the loop, so `offset + sizeof(ulong)` needs to re-evaluated, whilst `input.Length - sizeof(ulong)` can be evaluated only once and kept in a register and compared to another register value (`offset`).

Reply:
Thanks again! I wasn't really happy with that condition to start with, this makes it feel a bit better at the least.